### PR TITLE
New release: v4.7.0

### PR DESCRIPTION
**Pull request by lsm5**
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: version/rawversion/version.go [hunk 1]
```diff
@@ -7,4 +7,4 @@ package rawversion
 //
 // NOTE: remember to bump the version at the top of the top-level README.md
 // file when this is bumped.
-const RawVersion = "4.7.0"
+const RawVersion = "4.7.0-dev"
```
Comment (vrothberg) [inline version/rawversion/version.go:10]: `.1-dev`

Comment (lsm5) [reply version/rawversion/version.go:10]: ugh yes

Comment (lsm5) [reply version/rawversion/version.go]: done, thanks

**Review by vrothberg — APPROVED** [review]
/lgtm
/hold

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -97,6 +97,8 @@
 - Fixed a crash validating --device argument for create and run ([#19335](https://github.com/containers/podman/issues/19335)).
 - Fixed a bug where `.HostConfig.PublishAllPorts` always evaluates to `false` when inspecting a container created with `--publish-all`.
 - Fixed a bug in `podman image trust` command to allow using the local policy.json file ([#19073](https://github.com/containers/podman/issues/19073)).
+- Fixed a bug where a cgroup was not correctly mounted when running without a network namespace in rootless mode ([#20073](https://github.com/containers/podman/issues/20073)).
```
Comment (giuseppe) [inline RELEASE_NOTES.md:100]: I'd phrase it as "Fixed a bug where the cgroup file system was not correctly mounted inside the container when running without a network namespace in rootless mode"

Comment (lsm5) [reply RELEASE_NOTES.md]: done

**Comment by TomSweeneyRedHat** [comment]
Late on the review, but LGTM

**Pull request merged**